app/controllers/users_controller.rb

```diff
@@ -16,7 +14,19 @@ def create
 
   private
 
+  def user_from_params
+    email = user_params.delete(:email)
+    username = user_params.delete(:username)
+    password = user_params.delete(:password)
+
+    Clearance.configuration.user_model.new(user_params).tap do |user|
+      user.email = email
+      user.username = username
+      user.password = password
+    end
```

Comment:
I'm not sure why `email`, `username` and `password` are being assigned individually, while other fields are being assigned via `new(user_params)`. Is there something special about these three fields?